Tabspear/taofeekbadmus
# Complete the darkmode

## Changes

- Added hooks to change the background color of the page after its rendered.
- Added the styled components library, might remove it.


## Purpose
- This is to allow users to to toggle between light and dark modes 


## Approach

_How does this change address the requirement or problem?_

## Pre-Testing

- [ ] Activate seed user...
- [ ] Change config...

## Testing Steps

### To test _X_:

1. Start here click on the about page
2. Do this click on the button to toggle between the modes 


**Expected result**: The buttons should toggle between the modes 


Closes #00
